app/models/audio_file.rb

```diff
@@ -51,24 +63,13 @@ def update_version_status
 
   def set_status
     # only do template validations once the audio callback worker has succeeded
-    if status == UPLOADED
-      return
-    elsif status == NOTFOUND
-      self.status_message = "Audio file #{label} not found."
-      return
-    elsif status == FAILED
-      self.status_message = "Audio file #{label} failed to process."
-      return
-    end
-
-    template = audio_version.
-               try(:audio_version_template).
-               try(:audio_file_templates).
-               try(:find) { |aft| aft.position == position }
+    return if [UPLOADED, NOTFOUND, FAILED].include?(status)
```

Comment:
Note: the default `status_message`s are now set in the callback_worker instead of here.  (The worker should be the only place NOTFOUND or FAILED gets set anyways).